lacommunaute/templates/forum/partials/subcategory_forum_list.html

```diff
@@ -0,0 +1,37 @@
+<div id="subforums_area" class="container-fluid">
+    {% if tags_of_descendants %}
+        <div class="c-box">
+            <div class="container-fluid d-flex justify-content-center mb-3">Afficher les fiches contenant l'étiquette</div>
+            <div class="container-fluid d-flex justify-content-center">
+                {% include "forum/partials/forum_tags.html" with forum=forum tags=tags_of_descendants active_forum_tag_slug=active_forum_tag_slug only %}
+            </div>
+        </div>
+    {% endif %}
+    <div class="row mt-4" id="documentation-category-subforums">
+        {% for node in sub_forums.top_nodes %}
+            <div class="col-12 col-md-4 mb-3 mb-md-5">
+                <div class="card c-card has-one-link-inside h-100">
+                    {% if node.obj.image %}
+                        <div class="card-header card-header-thumbnail rounded">
+                            <img src="{{ node.obj.image.url }}" alt="{{ node.obj.name }}" class="img-fitcover img-fluid" loading="lazy" />
```

Comment:
Suggestion volée sur une relecture de mon PR sur les emplois - l'utilisation des attributes `height` et `width` vont empêcher que la page saute pendant le lazy loading

```suggestion
                            <img src="{{ node.obj.image.url }}" alt="{{ node.obj.name }}" class="img-fitcover img-fluid" loading="lazy" height="{{ node.obj.image.height }}" width="{{ node.obj.image.width }}" />
```

Reply:
A priori ça va poser problème avec les class .img-fluid et img-fit-cover

Reply:
Ces attributes ne changent pas du CSS, j'ai testé sur mon local et après l'image est téléchargée les bons styles de `img-fluid` et/ou `img-fit-cover` sont appliqués